options/index.html

```diff
@@ -5,55 +5,44 @@
-                <a class="left adblock-logo">
-                    <img src="/Icon.png">
-                    <span>CatBlock</span>
+                <a href="https://github.com/CatBlock/catblock" class="left adblock-logo" target="_blank">
+                    <img src="../Icon.png">
                 </a>
             </div>
-            <nav class="desktop">
-                <a data-scripts="/options/general.js" data-page="general" i18n="tabgeneral"></a>
-                <a data-scripts="/options/filters.js" data-page="filters" i18n="tabfilterlists"></a>
-                <a data-scripts="/options/catblock.js" data-page="catblock">CatBlock</a>
-                <a data-scripts="/filtering/domainset.js /filtering/filternormalizer.js /filtering/filtertypes.js /filtering/filteroptions.js /options/customize.js" data-page="customize" i18n="tabcustomize"></a>
-                <a data-scripts="/options/support.js" data-page="support" i18n="tabsupport"></a>
+            <nav id="desktopnav">
+                <a data-scripts="options/general.js" data-page="general" i18n="tabgeneral"></a>
+                <a data-scripts="options/filters.js" data-page="filters" i18n="tabfilterlists"></a>
+                <a data-scripts="options/catblock.js" data-page="catblock">CatBlock</a>
+                <a data-scripts="filtering/domainset.js filtering/filternormalizer.js filtering/filtertypes.js filtering/filteroptions.js options/customize.js" data-page="customize" i18n="tabcustomize"></a>
+                <a data-scripts="options/support.js" data-page="support" i18n="tabsupport"></a>
             </nav>
-            <nav class="mobile">
+            <nav id="mobilenav">
                 <button id="toggletabs"></button>
                 <div>
-                    <a data-scripts="/options/general.js" data-page="general" i18n="tabgeneral"></a>
-                    <a data-scripts="/options/filters.js" data-page="filters" i18n="tabfilterlists"></a>
-                    <a data-scripts="/options/catblock.js" data-page="catblock">CatBlock</a>
-                    <a data-scripts="/filtering/domainset.js /filtering/filternormalizer.js /filtering/filtertypes.js /filtering/filteroptions.js /options/customize.js" data-page="customize" i18n="tabcustomize"></a>
-                    <a data-scripts="/options/support.js" data-page="support" i18n="tabsupport"></a>
+                    <a data-scripts="options/general.js" data-page="general" i18n="tabgeneral"></a>
+                    <a data-scripts="options/filters.js" data-page="filters" i18n="tabfilterlists"></a>
+                    <a data-scripts="options/catblock.js" data-page="catblock">CatBlock</a>
+                    <a data-scripts="filtering/domainset.js filtering/filternormalizer.js filtering/filtertypes.js filtering/filteroptions.js options/customize.js" data-page="customize" i18n="tabcustomize"></a>
+                    <a data-scripts="options/support.js" data-page="support" i18n="tabsupport"></a>
```

Comment:
Why was the forward slash removed? Doesn't that ensure it goes to the root of the extension's file tree?